public class TestException{
    public static void main(String[] args){
        try{

        }catch(){

        }
    }
}
